examples/src/streaming/protocol.rs

```diff
@@ -0,0 +1,45 @@
+//! Protocol definitions for client-driven audio streaming
+
+// Protocol enum for current fastn-p2p API (will migrate to &'static str later - see GitHub issue #2)
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
+pub enum AudioProtocol {
+    GetInfo,
+    RequestChunk,
+}
+
+// Get stream metadata request/response
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub struct GetInfoRequest;
+
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub struct GetInfoResponse {
+    pub total_chunks: u64,
+    pub chunk_size_bytes: usize,
```

Comment:
Can we make chunk size variable? Server allows some window of sizes and client can request based on their bandwidth/RAM etc.